app/src/main/java/org/stepik/android/domain/certificates/repository/CertificatesRepository.kt

```diff
@@ -0,0 +1,9 @@
+package org.stepik.android.domain.certificates.repository
+
+import io.reactivex.Single
+import org.stepik.android.domain.base.DataSourceType
+import org.stepik.android.model.Certificate
+
+interface CertificatesRepository {
+    fun getCertificates(userId: Long, primarySourceType: DataSourceType = DataSourceType.REMOTE): Single<List<Certificate>>
```

Comment:
It seems that we never call `getCertificates` with `primarySourceType = CACHE`, also it shouldn't be called `primary` as `CertificatesRepositoryImpl` doesn't contain fallback logic